src/gmt_remote.c

```diff
@@ -257,9 +267,22 @@ GMT_LOCAL struct GMT_DATA_INFO *gmtremote_data_load (struct GMTAPI_CTRL *API, in
 		return NULL;
 	}
 
+	/* Watch for "#% " records with high-res data in very odd resolutions, such as 52.0732883317s for Pluto's original grid.
```

Comment:
Please see https://github.com/GenericMappingTools/gmtserver-admin/issues/171#issuecomment-1720523604.

Reply:
Yep, after dinner etc.

Reply:
Hi @seisman, while not tested yet, see if you think the changes in gmt_remote.c around lines 280-288 should fix the problem, tighter with the changes in gmtserver-admin that should no longer count the #% data comments in the initial number given in record 1.  The idea is that GTM 6.5 and later increment the total number by 1 for each such data set parsed.

Reply:
I haven't tried this PR yet, but my main point in https://github.com/GenericMappingTools/gmtserver-admin/issues/171#issuecomment-1720523604 is that, if we don't make any changes to the oceanic server (i.e., adding the new planetary dataset and don't use the magic `#%` marker), both GMT 6.4 and 6.5 still work except that GMT 6.4 crashes if someone tries to access the full pluto/mars grids with GMT 6.4. 

Maybe we don't have to introduce the magic `#%` marker and simply make changes to the remote-dataset docs and acknowledge that the full-resolution pluto/mars grids requires GMT>=6.5.

Reply:
I will test this before any 6.5 and data release.  I think the implemented solution (if working) is the right way:

1.<= 6.4 versions will read the correct number in record one (it won't contain the two odd datasets in the count) and it will skip those #% records as comments.
2 >= 6.5 will red the same (lower) count but then add 1 for each #% record processed.

The result is that GMT 6.5 and future will enjoy all those data sets while older GMT will enjoy all the new files except those two.  O think that is a good solution, no? The scripts/makefile for gmtserver-admin have been adjusted to not include the #% record in the data count and the GMT C code is expecting this.  I will of course test both the scripts and C code with both 6.4 and 6.5.